test/functional/specs/desktop/C2592.js

```diff
@@ -0,0 +1,33 @@
+import { t, Selector } from "testcafe";
+import createNetworkLogger from "../../src/networkLogger";
+import { responseStatus } from "../../src/assertions/index";
+import fixtureFactory from "../../src/fixtureFactory";
+
+const urlCollector = `http://127.0.0.1:8080/test/functional/sandbox/html/alloySdk.html`;
```

Comment:
I added this constant in a different test:
https://github.com/adobe/alloy/blob/master/test/functional/src/constants/testServerUrl.js